activities/communication/run-a-community-call.md

```diff
@@ -0,0 +1,109 @@
+---
+type: Resource
+---
+
+# How to run a community call
+
+The roles on the community call are:
+
+* the chair, who opens the call on behalf of the Foundation
+* experts leading specific substantive sections of the call (otherwise this is the chair)
+* the notetaker
+
+## Before the call
+
+The chair and the notetaker:
+
+* create a notes HackMD.io document for this call
+* add "updates since the last community call" to the notes - keep this to three items or less (we want the community call to be a dialogue, not a broadcast). If possible, focus on community related updates
+* put these in the call notes doc
+
+Ask the [communications coordinator](../../organization/staff.md#communications) if you need help with Foundation updates.
+
+The notetaker will take notes during the call in the same document.
+
+### Conference call etiquette
+
+Each person should use their own computer so everyone on the call is symbolically equal.
+
+The chair should be on the call 5 minutes before the call starting time.
+
+Consider putting a Foundation for Public Code poster on the wall behind you.
+
+## Templated agenda items
+
+1. Opening the call
+
+* Introduction round (all)
+* Short update from the Foundation for Public Code (chair)
+
+2. Substance of the call
```

Comment:
I believe this is invalid markdown and might create unexpected results. The separate `2.` might be taken, because it is in a new paragraph, to be a new ordered list and might render in the browser as `1.`.

I'd suggest replacing it with this:

```markdown
1. Opening the call
  * Introduction round (all)
  * Short update from the Foundation for Public Code (chair)
2. Substance of the call
```